src/calculateRentalCost.js

```diff
@@ -4,7 +4,25 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const basePrice = 40;
+
+  if (days === 1) {
+    return basePrice;
+  }
+
+  if (days > 1 && days < 3) {
+    return basePrice * days;
+  }
+
+  if (days >= 3 && days <= 6) {
+    return basePrice * days - 20;
+  }
+
+  if (days > 6) {
+    return basePrice * days - 50;
+  }
+
+  return basePrice;
```

Comment:
This return statement is redundant because all possible cases for the number of days are already covered by the preceding if-else conditions. You can safely remove this line.